opm/material/fluidsystems/BrineCO2FluidSystem.hpp

```diff
@@ -51,10 +52,10 @@ namespace Opm {
 
 // Silence compiler warnings about use of variables
 // that are instantiated in a different compilation unit.
-template<>
-const float CO2<float>::brineSalinity;
-template<>
-const double CO2<double>::brineSalinity;
+// template<>
```

Comment:
Why is this removed?

Reply:
I have just set the value inside the CO2 class, I dont think these lines are needed now

Reply:
I'll readd them iff the warnings are still there in jenkins